.github/workflows/iwyu.yml

```diff
@@ -0,0 +1,145 @@
+           libvulkan-dev                                      \
+           libopengl-dev                                      \
+           libglx-dev                                         \
+           mesa-common-dev                                    \
+           qtbase5-dev                                        \
+           libgtest-dev                                       \
+           libgmock-dev                                       \
+           git                                                \
+           ninja-build                                        \
+           clang-format-14                                    \
+           llvm-15-dev                                        \
+           libclang-15-dev                                    \
+           clang-15                                           \
+           patchutils                                         \
+           libprotobuf-dev
+      - name: Checkout IWYU
+        uses: actions/checkout@v3
+        with:
+          repository: include-what-you-use/include-what-you-use
+          ref: clang_15
+          path: iwyu
+      - run: mkdir build
+        working-directory: ./iwyu
+      - name: Configure IWYU
+        working-directory: ./iwyu/build
+        run: |
+          cmake -G Ninja -DCMAKE_BUILD_TYPE=Release -DCMAKE_PREFIX_PATH=/usr/lib/llvm-15 -DCMAKE_INSTALL_PREFIX=/usr "${GITHUB_WORKSPACE}/iwyu"
+      - name: Build and install IWYU
+        working-directory: ./iwyu/build
+        run: sudo cmake --build "${GITHUB_WORKSPACE}/iwyu/build" --target install
+      - run: mkdir build
+        working-directory: ./orbit
+      - name: CMake Configure
+        working-directory: ./orbit/build
+        run: |
+          cmake -DCMAKE_BUILD_TYPE=Release                                                             \
+           -DCMAKE_CXX_FLAGS="-march=sandybridge -fno-omit-frame-pointer -mno-omit-leaf-frame-pointer" \
+           "${GITHUB_WORKSPACE}/orbit"
+      - name: CMake Build
+        working-directory: ./orbit/build
+        run: cmake --build . --target iwyu
+      - name: Try to apply changes
+        working-directory: ./orbit
+        run: |
+          git checkout -b "hotfix/iwyu-${GITHUB_SHA}"
+          git update-index --refresh
+          git apply --index build/iwyu.diff
+      - name: Check for changes
+        working-directory: ./orbit
+        id: check_changes
+        run: |
+          if [[ -z "$(git diff --cached --exit-code)" ]]; then
+            echo "has_changes=false" >> $GITHUB_OUTPUT
+          else
+            echo "has_changes=true" >> $GITHUB_OUTPUT
+          fi
+      - name: Apply and push changes
+        if: steps.check_changes.outputs.has_changes == 'true'
+        working-directory: ./orbit
+        run: |
+          git commit --message="Automatically apply IWYU to the codebase"
+          git push --set-upstream origin "hotfix/iwyu-${GITHUB_SHA}"
```

Comment:
Nit: Could we put the branch name into a variable?

Reply:
Done.